formatTest/typeCheckedTests/expected_output/arityConversion.re

```diff
@@ -16,11 +16,11 @@ module Test = {
     | Or (int, int);
 };
 
-Test.And (1, 2);
+let _ = Test.And (1, 2);
```

Comment:
Is it the case that there is loss right now when converting from reason to reason, since the input file has `Test.And (1, 2);` yet the output has `let _ = `?